doc/api/v8.md

```diff
@@ -308,6 +320,7 @@ changes:
   generated, where `{pid}` will be the PID of the Node.js process,
   `{thread_id}` will be `0` when `writeHeapSnapshot()` is called from
   the main Node.js thread or the id of a worker thread.
+* `options` {Object} See [`v8.getHeapSnapshot()`][] for more details.
```

Comment:
The options are simple enough that I would just duplicate the listing here rather than send folks off to some other place.